apps/web-client/src/lib/stores/inventory/__tests__/table_content.test.ts

```diff
@@ -1,15 +1,20 @@
 import { describe, test, expect } from 'vitest';
+import { testUtils } from '@librocco/shared';
 
-import type { PaginationData } from '$lib/types/inventory';
+import type { PaginationData, DisplayRow } from '$lib/types/inventory';
 
 import { createDisplayEntriesStore, createPaginationDataStore } from '../table_content';
 
 import type { VolumeQuantity } from '$lib/types/db';
 import { readable } from 'svelte/store';
+import { newTestDB } from '$lib/__testUtils__/db';
+const { waitFor } = testUtils;
 
 describe('tableContentStore', () => {
 	test("should not explode if no 'entity' is provided", async () => {
-		const de$ = createDisplayEntriesStore(undefined, readable(0), readable({}), {});
+		const db = await newTestDB();
+
+		const de$ = createDisplayEntriesStore(db, undefined, readable(0), {});
```

Comment:
This should also work if `db` is undefined (should have the same behaviour as if the entiry is not defined)

Reply:
I would prefer to test for both cases:
1. If both are undefined
2. If db is defined, but not the entity
 _(we won't have a case where db is undefined, and entity is)_

Reply:
This could be a single `test()` case ☝️